src/Object.h

```diff
@@ -45,6 +45,11 @@ namespace sdbus::internal {
     public:
         Object(sdbus::internal::IConnection& connection, std::string objectPath);
 
+        const std::string& getObjectPath() const override
+        {
+            return objectPath_;
+        }
```

Comment:
Additionally, would you please declare the method here and implement it in the `.cpp` file? The method is virtual, so inlining it doesn't truly make sense, the method will be instantiated in several object files and then the linker will anyway have to pick one single place. And anyway, defining it in the `.cpp` file will be consistent with other interface, virtual methods.

Reply:
Done